concent_api/utils/logging.py

```diff
@@ -340,7 +346,7 @@ def log_json_message(logger: Logger, message: JsonResponse):
 
 
 def log_string_message(logger: Logger, *messages_to_log: str):
-    logger.info(join_messages(messages_to_log))
+    logger.info(' '.join(messages_to_log))
```

Comment:
Why not `join_messages()`? If you're sure that none of the messages is `None` or `''` please add an assertion checking that. Otherwise you should use `join_messages()`.

Reply:
`join_messages` returns string like `('message1', 'message2')`. It does not look good in logs, so i changed it to simple `join`, instead of use 'join_messages` and format output. Added assertion.

Reply:
It works good.. I should just use it like `join_messages(*messages_to_log)` instead of `join_messages(messages_to_log)`. It is merged now, will be fixed in separate pull request